src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,107 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
+
+    private int hash(K key) {
+        int index = key.hashCode() & (table.length - 1);
```

Comment:
do you know what does `&` mean?

Reply:
yes, I can write Math.abs(key.hashCode() % (table.length - 1)) and get the same result. Math.abs means that number will be positive. I chose &, because i saw this realization in this article - https://habr.com/ru/post/421179/